diff --git a/fltk/Fl_Scintilla.h b/fltk/Fl_Scintilla.h
index c93cc07..f10fd48 100644
--- a/fltk/Fl_Scintilla.h
+++ b/fltk/Fl_Scintilla.h
@@ -93,7 +93,7 @@ public:
 		return ScintillaBase::WndProc(iMessage, wParam, lParam);
 	}
 
-private:
+protected:
 	struct {
 		void (*callback)(Scintilla::SCNotification *scn, void *data);
 		void *data;
diff --git a/src/ExternalLexer.h b/src/ExternalLexer.h
index 89cf024..609b179 100644
--- a/src/ExternalLexer.h
+++ b/src/ExternalLexer.h
@@ -8,20 +8,14 @@
 #ifndef EXTERNALLEXER_H
 #define EXTERNALLEXER_H
 
-#if PLAT_WIN
-#define EXT_LEXER_DECL __stdcall
-#else
-#define EXT_LEXER_DECL
-#endif
-
 #ifdef SCI_NAMESPACE
 namespace Scintilla {
 #endif
 
-typedef void*(EXT_LEXER_DECL *GetLexerFunction)(unsigned int Index);
-typedef int (EXT_LEXER_DECL *GetLexerCountFn)();
-typedef void (EXT_LEXER_DECL *GetLexerNameFn)(unsigned int Index, char *name, int buflength);
-typedef LexerFactoryFunction(EXT_LEXER_DECL *GetLexerFactoryFunction)(unsigned int Index);
+typedef void*(*GetLexerFunction)(unsigned int Index);
+typedef int (*GetLexerCountFn)();
+typedef void (*GetLexerNameFn)(unsigned int Index, char *name, int buflength);
+typedef LexerFactoryFunction(*GetLexerFactoryFunction)(unsigned int Index);
 
 /// Sub-class of LexerModule to use an external lexer.
 class ExternalLexerModule : public LexerModule {
